Tasks/DownloadBuildArtifact/task.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "a433f589-fce1-4460-9ee6-44a624aeb1fb",
+  "name": "DownloadBuildArtifact",
+  "friendlyName": "Download Build Artifact",
+  "description": "Download a build artifact.",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 48
```

Comment:
I was iterating.  Does it matter where we start it?  I can bump it down to 0.0.1 if it's really important.